src/test-utils/fetch-mock.ts

```diff
@@ -0,0 +1,5 @@
+import fetch from 'node-fetch'
+import fetchMockJest from 'fetch-mock-jest'
+const fetchMock = fetch as unknown as typeof fetchMockJest
+
+export default fetchMock
```

Comment:
These type shenanigans are necessary since we don't have a global fetch and have to use the `node-fetch` import instead of `fetch-mock` itself